theano/tensor/tests/test_opt.py

```diff
@@ -3567,6 +3569,7 @@ def test_equality_shapes(self):
         f = theano.function([x], T.eq(g, 0))
         assert (f([3, 3]) == 0).all()
         assert (f([]) == 1).all()
+        self.assertTrue(check_stack_trace(f, ops_to_check='last'))
```

Comment:
Same here, which optimization gets applied, and what is supposed to be the last node?